keprekar's constant/kaprekarsConstant.js

```diff
@@ -0,0 +1,59 @@
+const { LENGTH_ERROR, DIGIT_ERROR, FINAL_RESULT, KEPREKARS_CONSTANT } = require("./labelConstants");
+
+export const generateRandomInput = () => {
+    const result = Math.floor(1000 + Math.random() * 9000);
+    console.log("Initial Number:", result);
```

Comment:
Side Effects should be avoided. You should have consoled the initial number in the "karekarsRoutine" function